eng/targets/ReferenceAssembly.targets

```diff
@@ -0,0 +1,93 @@
+<Project>
+
+  <Target Name="GenerateReferenceSource" Condition="'$(HasReferenceAssembly)' == 'true'">
+    <PropertyGroup>
+      <_RefSourceOutputPath>$([System.IO.Directory]::GetParent('$(MSBuildProjectDirectory)'))/ref/</_RefSourceOutputPath>
+      <_RefProjectFileOutputPath>$(_RefSourceOutputPath)$(AssemblyName).csproj</_RefProjectFileOutputPath>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_AllTargetFrameworks Remove="@(_AllTargetFrameworks)" />
+      <_AllTargetFrameworks Include="$(TargetFrameworks);$(TargetFramework)" />
+      <_AllTargetFrameworks Remove="net46;net461;net471;net472" />
```

Comment:
Are you trying to disable reference generation for .NET Framework? If so, we could do that for all versions by conditioning the _GenerateProjectSourceInner target on `'$(TargetFrameworkIdentifier)' != '.NETFramework'`

Reply:
It's also used in file generation but I think I know how to workaround it.